code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -91,6 +94,18 @@
 				return
 			accesses -= get_region_accesses(region)
 			. = TRUE
+		if("close_speed")
+			var/tune = params["tune"]
+			var/adjust = text2num(params["adjust"])
+			if(adjust)
+				tune = close_speed + adjust * 10
+				. = TRUE
+			else if(text2num(tune) != null)
+				tune = tune * 10
```

Comment:
```suggestion
				tune = close_speed + adjust SECONDS
				. = TRUE
			else if(text2num(tune) != null)
				tune = tune SECONDS
```
also, I'm assuming stuff done with "tune" is WIP?

Reply:
its what the radio freq stuff did and it seemed to work so.